migration/src/main/java/uk/ac/bbsrc/tgac/miso/migration/destination/MisoServiceManager.java

```diff
@@ -103,7 +105,8 @@
   private SQLPoolDAO poolDao;
   private SQLPoolQCDAO poolQcDao;
   private SQLExperimentDAO experimentDao;
-  private SQLKitDAO kitDao;
+  private SQLKitComponentDAO kitDao;
```

Comment:
Please rename to `kitComponentDao`, and update getters/setters for DAO similarly.